api/types/app.go

```diff
@@ -25,9 +25,14 @@ import (
 	"github.com/gravitational/trace"
 
 	"github.com/gravitational/teleport/api/constants"
+	"github.com/gravitational/teleport/api/types/compare"
 	"github.com/gravitational/teleport/api/utils"
 )
 
+var (
+	_ compare.IsEqual[Application] = (*AppV3)(nil)
+)
```

Comment:
nit:

```suggestion
var _ compare.IsEqual[Application] = (*AppV3)(nil)
```

So on for all single-element var blocks.